docs/development/translations.rst

```diff
@@ -0,0 +1,397 @@
+Translations
+============
+
+Quick Start Guide
+-----------------
+
+SecureDrop is a system that lets people share sensitive information
+with investigative journalists anonymously and securely.
+Learn :ref:`more about SecureDrop <what_is_securedrop>`.
+
+SecureDrop is written in English and translated into multiple
+other languages. Translations are managed using **Weblate**,
+a web platform that enables collaborative translation projects.
+
+.. _get_started_using_weblate:
+
+Get started using Weblate
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can choose to :ref:`register on Weblate <how_to_register_on_weblate_email>`
+with your email address, or by linking a GitHub account.
+
+You can contribute to any language, and Weblate has some conveniences
+to make it easier to work with your preferred languages.
+Learn :ref:`how to choose your preferred languages on Weblate <how_to_manage_your_preferred_languages_weblate>`.
+
+Our Weblate instance only contains one project, SecureDrop, which has
+two translation components: the main SecureDrop web application (labeled
+"SecureDrop"), and the translations for the desktop icons of the admin
+and journalist workstations used by news organizations (labeled "desktop").
+
+.. figure:: ../images/weblate/project.png
+   :align: center
+   :figwidth: 80%
+   :width: 100%
+
+   Components of the SecureDrop project in Weblate. Click on the image to see it full-size.
+
+Once a string is translated, it must be approved by a :ref:`reviewer <weblate_glossary_reviewer>`
+before being accepted into SecureDrop.
+
+Approved strings can only be modified by reviewers.
+However, translators can still suggest modifications and make comments
+if they notice something wrong.
+SecureDrop translations are a collaborative endeavour!
```

Comment:
The old document [mentioned how to become a reviewer](https://github.com/freedomofpress/securedrop-docs/blob/main/docs/development/l10n.rst#reviews) (by asking in the forum). This would be a good place to do that, though maybe we should prefer Gitter at this point.

Reply:
I'm thinking of placing a sentence here "Learn how to become a reviewer" with a link to a how-to (I have the previous draft handy) then adding that same link at the end of the "reviewer" entry in the glossary.

Reply:
I added a new **How to become a reviewer** and linked it from the **Reviewer** glossary entry. The instructions currently link to the forum because that's also where the instructions to add a language point to, I've no preference on that but went for consistency.

On second thought, and looking at the surrounding context in **Getting started using Weblate**, I think a link there about "how to become a reviewer" would risk feeling a little out of place, or maybe unnecessary, considering that: 

- the section aims at getting people started and the other link is "Learn how to translate SecureDrop using Weblate." It seems to me that learning to translate is a necessary step before becoming a reviewer, isn't it?
- the first mention of "reviewer" links to the glossary entry where the link to the how-to is unmissable (Which means that if becoming a reviewer is that's what you've in mind, the how-to is in the table of contents or one click away.)

What do you think @rmol?

Reply:
Yep, good points. I think this is ready.

Reply:
Do you want me to rebase the PR @rmol? I think it's got a little bit behind `main` since last time I did it, and I could squash the two new commits as well.

Reply:
Oops, didn't see this before merging. I think it's fine.